packages/dds/merge-tree/src/test/client.searchForMarker.spec.ts

```diff
@@ -166,253 +113,256 @@ describe("TestClient", () => {
+				[reservedTileLabelsKey]: [markerLabel],
 				[reservedMarkerIdKey]: "some-id",
 			});
 
 			client.insertTextLocal(0, "abc d");
 
 			client.insertMarkerLocal(0, ReferenceType.Tile, {
-				[reservedTileLabelsKey]: [tileLabel],
+				[reservedTileLabelsKey]: [markerLabel],
 				[reservedMarkerIdKey]: "some-id",
 			});
 
 			client.insertTextLocal(7, "ef");
 			client.insertMarkerLocal(8, ReferenceType.Tile, {
-				[reservedTileLabelsKey]: [tileLabel],
+				[reservedTileLabelsKey]: [markerLabel],
 				[reservedMarkerIdKey]: "some-id",
 			});
 			console.log(client.getText());
 
 			assert.equal(client.getLength(), 10, "length not expected");
 
-			const tile = client.searchForMarker(5, tileLabel, false);
+			const marker = client.searchForMarker(5, markerLabel, false);
 
-			assert(tile, "Returned tile undefined.");
+			assert(marker, "Returned marker undefined.");
 
 			const exp = client.mergeTree.referencePositionToLocalPosition(
-				tile,
+				marker,
 				UniversalSequenceNumber,
 				client.getClientId(),
 			);
 
-			assert.equal(exp, 0, "Tile with label not at expected position");
+			assert.equal(exp, 0, "Marker with label not at expected position");
 		});
 
-		it("Should be able to find non preceding tile position from client with multiple tile", () => {
-			const tileLabel = "EOP";
+		it("Should be able to find non preceding marker position from client with multiple marker", () => {
+			const markerLabel = "EOP";
 			client.insertMarkerLocal(0, ReferenceType.Tile, {
-				[reservedTileLabelsKey]: [tileLabel],
+				[reservedTileLabelsKey]: [markerLabel],
 				[reservedMarkerIdKey]: "some-id",
 			});
 
 			client.insertTextLocal(0, "abc d");
 
 			client.insertMarkerLocal(0, ReferenceType.Tile, {
-				[reservedTileLabelsKey]: [tileLabel],
+				[reservedTileLabelsKey]: [markerLabel],
 				[reservedMarkerIdKey]: "some-id",
 			});
 
 			client.insertTextLocal(7, "ef");
 			client.insertMarkerLocal(8, ReferenceType.Tile, {
-				[reservedTileLabelsKey]: [tileLabel],
+				[reservedTileLabelsKey]: [markerLabel],
 				[reservedMarkerIdKey]: "some-id",
 			});
 			console.log(client.getText());
 
 			assert.equal(client.getLength(), 10, "length not expected");
 
-			const tile = client.searchForMarker(5, tileLabel, true);
+			const marker = client.searchForMarker(5, markerLabel, true);
 
-			assert(tile, "Returned tile undefined.");
+			assert(marker, "Returned marker undefined.");
 
 			const exp = client.mergeTree.referencePositionToLocalPosition(
-				tile,
+				marker,
 				UniversalSequenceNumber,
 				client.getClientId(),
 			);
 
-			assert.equal(exp, 6, "Tile with label not at expected position");
+			assert.equal(exp, 6, "Marker with label not at expected position");
 		});
 
-		it("Should be able to find non preceding tile with multiple segments and tiles", () => {
-			const tileLabel = "EOP";
+		it("Should be able to find non preceding marker with multiple segments and markers", () => {
```

Comment:
is there a test analogous to this but where the markers are sparse in the tree? E.g. multi-level tree with only a couple markers near the end, starting a search near 0 and getting correct results?